daemon/cmd/cells.go

```diff
@@ -244,6 +245,9 @@ var (
 
 		// The device reloader reloads the datapath when the devices change at runtime.
 		cell.Invoke(registerDeviceReloader),
+
+		// The node discovery cell provides the local node configuration and node discovery.
```

Comment:
Perhaps could spell out what we mean by "node discovery" here, e.g. it propagates changes in the local node towards other Cilium nodes via CiliumNode CRD and kvstore.